Update README.md

benallfree opened the pull request:
Clarification about how to get `useHooks` error to go away.

martpie commented:
Is this how you fixed it? Can you share the `package.json` of your different packages? (I'm interested in the solution as I could no manage to get it working myself)

benallfree commented:
Sure. This solution is specific to yarn 1.x (classic) workspaces, but if it helps anyone I'm happy to share more :)


```
/package.json
----------------
{
  "name": "workspace",
  "version": "0.0.1",
  "private": true,
  "workspaces": [
    "packages/*"
  ]
}



/packages/library/package.json
-----------------
{
  "name": "library",
  "version": "0.0.1",
  "peerDependencies": {
    "react": "^16.13.1",
    "react-dom": "^16.13.1"
  }
}

/packages/next-app/package.json
------------------------------------
{
  "name": "next-app",
  "version": "0.0.1",
  "dependencies": {
    "next": "9.3.5",
    "react": "16.13.1",
    "react-dom": "16.13.1"
  }
}

```

martpie commented:
Thanks!

martpie commented:
If someone sees this thread: please note it is mostly directed to Yarn. I think `npm` still has this issue unfortunately.

benallfree commented:
It might be worth noting that I didn't even need the webpack customization when using yarn. Once I moved `react` to `peerDependencies`, everything was happy. As you say `npm` might be different. I updated my comment above with the `next.config.js` sample that works too.
